crates/common/src/types.rs

```diff
@@ -149,11 +149,11 @@ impl ToAlloy for ethers_core::types::Transaction {
     }
 }
 
-impl ToReth for alloy_rpc_types::trace::parity::LocalizedTransactionTrace {
-    type To = alloy_rpc_types::trace::parity::LocalizedTransactionTrace;
+impl ToReth for alloy_rpc_trace_types::parity::LocalizedTransactionTrace {
```

Comment:
this is now redundant?

Reply:
true! we can remove this

Reply:
removed the `ToReth` trait entirely in https://github.com/foundry-rs/foundry/pull/6705/commits/d8f44c15567cba5be55cdefee0b1c483ee8c798c